### PR TITLE
Update SearchQueue.js

### PR DESCRIPTION
**Pull request by idanb11**
remove redundant logging...

### CODE REVIEW

File: lib/SearchQueue.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ SearchQueue.prototype = {
          queryObj = queryObj.hasOwnProperty('query') ? queryObj : { "query": queryObj };
          this.esc.search(dat.index, dat.type, queryObj, dat.options)
             .on('data', function(data) {
-               console.log('search result', data);
+               //console.log('search result', data);
```
Comment (katowulf) [inline lib/SearchQueue.js]: Please delete instead of commenting.

**Comment by katowulf** [comment]
@idanb11 thanks for the fixes! Please address this one comment and we'll get merged.

**Comment by idanb11** [comment]
Done.

**Comment by katowulf** [comment]
There are conflicts so it won't let me merge : (  Sorry for the additional hurdle. Thanks again for the contribution!

**Comment by idanb11** [comment]
created a new PR #47

**Pull request closed without merging**